mirage-xen-ocaml.opam

```diff
@@ -15,4 +15,4 @@ depends: [
   "ocaml-src"
   "ocamlbuild" {build}
 ]
-available: [ocaml-version >= "4.04.2" & ocaml-version <= "4.05.0" & os = "linux"]
+available: [ocaml-version >= "4.04.2" & ocaml-version <= "4.06.0" & os = "linux"]
```

Comment:
this is not correct -- it should be `< "4.06.0"` or `<= "4.05.0"`

Reply:
Oh, indeed, thanks.